src/logic/function/basic.lean

```diff
@@ -432,6 +432,13 @@ lemma comp_update {α' : Sort*} {β : Sort*} (f : α' → β) (g : α → α') (
   f ∘ (update g i v) = update (f ∘ g) i (f v) :=
 funext $ apply_update _ _ _ _
 
```

Comment:
This lemma seems odd I'm the absence of a `single_op`. Do we already have that lemma under a different name?

Reply:
Removed this lemma.